vayesta/core/fragmentation/fragmentation.py

```diff
@@ -195,7 +195,7 @@ def rotational_symmetry(self, order, axis, center=(0,0,0), unit='Ang'):
         self.sym_register.append([])
         yield
         fragments = self.sym_register.pop()
-        fragments_sym = self.emb.create_rotsym_fragments(order, axis, center, fragments=fragments, unit=unit,
+        fragments_sym = self.emb.create_rotsym_fragments(order, tuple(axis), tuple(center), fragments=fragments, unit=unit,
```

Comment:
This change is why `xyz` are no longer being accepted as valid axis options; they're being converted into tuples `("x",)` and no longer recognised later on. Why are we adding the tuple requirement here?

Reply:
Fix which permits the use of lists to specify symmetry axes pushed.